CreateBootstrappedMSBuild.cmd

```diff
@@ -16,15 +15,14 @@ if not defined VS140COMNTOOLS (
 )
 
 if not "%1"=="" (
-    set OutputPath=%1
+    set AdditionalBuildCommand=%1
 )
 
 echo ** Building with the installed MSBuild
-echo ** Output Path: %DebugBuildOutputPath%
-echo ** Additional Build Parameters:%AdditionalBuildCommand%
+echo ** Additional Build Parameters: %AdditionalBuildCommand%
 echo.
 :: Build MSBuild
-call "%~dp0build.cmd" /t:Rebuild %AdditionalBuildCommand%
+call "%~dp0build.cmd" /t:Rebuild /p:Platform=x86 %AdditionalBuildCommand%
```

Comment:
I didn't do this because it doesn't really make sense to bootstrap x64 like this. I want to convert the bootstrap to build both platforms and deploy x64 to the amd64 sub-folder, but maybe next time.